examples/ownership/src/main.sw

```diff
@@ -0,0 +1,31 @@
+contract;
+
+use std::{identity::Identity, option::Option};
+
+abi OwnershipExample {
+    #[storage(write)]fn revoke_ownership();
+    #[storage(write)]fn set_owner(identity: Identity);
+    #[storage(read)]fn owner() -> Option<Identity>;
+}
+
+storage {
+    owner: Option<Identity>,
+}
+
+impl OwnershipExample for Contract {
+    // ANCHOR: revoke_owner_example
+    #[storage(write)]fn revoke_ownership() {
+        storage.owner = Option::None();
+    }
+    // ANCHOR_END: revoke_owner_example
+
+    // ANCHOR: set_owner_example
```

Comment:
This anchor doesn't seem to be used. Are you planning on using it eventually?

Reply:
Originally I didn't intend to use it and left it for future use. However, with my most recent update I am now using it.